htdocs/compta/bank/class/account.class.php

```diff
@@ -54,14 +55,6 @@ class Account extends CommonObject
     var $clos;
     var $rappro=1;    // If bank need to be conciliated
     var $url;
-    //! BBAN field for French Code banque
-    var $code_banque;
```

Comment:
Your CCC system is like RIB system, your control code is before the account number, for us, in france, it's after the account number.

Normally, this system is deprecated more than 1 year when this version was released in stable version.